bin/propolis-utils/README.md

```diff
@@ -0,0 +1,11 @@
+# Propolis Utilities
+
+This is a collection of assorted utilities which may be useful for certain
+development activities, but are otherwise not meant for general consumption.
+
+## Utilities
+
+- `savex`:  Extract data fields from a `propolis-standalone` instance which was
+  halted and saved with the `-s` flag.
+- `cpuid-gen`: Generated CPUID profile using the legacy emulated output from the
+  local host CPU, as filtered by the kernel VMM logic.
```

Comment:
:o cool addition!

Reply:
Once CPUID handling gets wired up, I wanted a util to more easily generate profiles for hardware (such as Intel ;-) ) which differs from what we're shipping in production, but is still necessary for dev work.